Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -9752,6 +9753,44 @@ void WebPage::hasActiveNowPlayingSessionChanged(bool hasActiveNowPlayingSession)
     send(Messages::WebPageProxy::HasActiveNowPlayingSessionChanged(hasActiveNowPlayingSession));
 }
 
+void WebPage::getFrameOwnerPermissionsPolicy(WebCore::FrameIdentifier frameID, CompletionHandler<void(const WebCore::OwnerPermissionsPolicyData&)>&& completionHandler)
+{
+    WebCore::OwnerPermissionsPolicyData policy;
```

Comment:
I would tend to move it where we actually set members. Maybe we could use a constructor.
And use `completionHandler({ })` for the error cases.